libnetwork/pasta/pasta_linux.go

```diff
@@ -217,8 +217,12 @@ func createPastaArgs(opts *SetupOptions) ([]string, []string, error) {
 		cmdArgs = append(cmdArgs, "--no-map-gw")
 	}
 
-	// always pass --quiet to silence the info output from pasta
-	cmdArgs = append(cmdArgs, "--quiet", "--netns", opts.Netns)
+	if !slices.Contains(cmdArgs, "--trace") && !slices.Contains(cmdArgs, "--debug") {
```

Comment:
we already iterate the slice on line 151 so I suggest you add the check there and store a bool like we do for the other options.

Also please add a new unit test case in pasta here https://github.com/containers/common/blob/116500d0a7b50f9d2cbf46b0c9ab543e22d844b8/libnetwork/pasta/pasta_linux_test.go#L21 to make sure it works as expected and we do not regress in the future

Reply:
Hello
* I add the check on line 151
* I add a test case